components/tabs/src/fixtures.js

```diff
@@ -226,16 +238,16 @@ const arrProposedBabel = [
 
 /* eslint-disable-next-line react/no-multi-comp */
 class ProposedClassPropertiesPlugin extends Component {
-  static defaultProps = sharedDefaultProps;
-
-  static propTypes = sharedPropTypes;
-
   state = { tabIndex: this.props.defaultIndex };
 
   setTabIndex = setTabIndex;
 
   handleClick = handleClick;
 
+  static propTypes = sharedPropTypes;
```

Comment:
why move these?

Reply:
linting was saying that `propTypes` had to go after `handleClick`, and the `defaultProps` after `propTypes`.

we seem to have ended up with slightly stricter linting rules with the updated yarn.lock file